mon-pix/app/routes/campaigns/assessment/skill-review.js

```diff
@@ -20,6 +20,10 @@ export default class SkillReviewRoute extends Route {
         userId: user.id,
       });
       const trainings = await campaignParticipation.hasMany('trainings').reload();
+
+      // Reload the user to display my trainings link on the navbar menu
+      if (trainings?.length > 0 && !(await user.hasRecommendedTrainings)) await this.currentUser.load();
```

Comment:
```suggestion
      if (trainings?.length > 0 && !user.hasRecommendedTrainings) await this.currentUser.load();
```
pas besoin du await à mon sens

Reply:
IMO les `{` `}` ne coutent rien et précisent où la condition s'applique